packages/riverpod/lib/src/core/override.dart

```diff
@@ -30,9 +37,14 @@ extension on _ProviderOverride {
   }
 }
 
-sealed class _FamilyOverride implements Override {}
+/// An object used by [ProviderContainer]/`ProviderScope` to override the behavior
+/// of a family for part of the application.
+@visibleForTesting
+sealed class FamilyOverride implements Override {}
 
-extension on _FamilyOverride {
+/// Extension methods for [FamilyOverride].
+@visibleForTesting
+extension FamilyOverrideX on FamilyOverride {
```

Comment:
There's no point in exposing `from`. For starting, it's accesible from the `ProviderBase` interface